apps/finance/app/src/transfer-types.js

```diff
@@ -2,11 +2,16 @@ export const All = Symbol('ALL_TRANSFER')
 export const Incoming = Symbol('INCOMING_TRANSFER')
 export const Outgoing = Symbol('OUTGOING_TRANSFER')
 
-export const READABLE_TRANSFER_TYPES = {
+const READABLE_TRANSFER_TYPES = {
   [All]: 'All',
   [Incoming]: 'Incoming',
   [Outgoing]: 'Outgoing',
 }
 
-export const TRANSFER_TYPES = Object.keys(READABLE_TRANSFER_TYPES)
-export const TRANSFER_TYPES_LABELS = Object.values(READABLE_TRANSFER_TYPES)
+// As we're using symbols as object keys,
+// we need to use Reflect.ownKeys to properly retrieve and iterate
+// over the object values.
+export const TRANSFER_TYPES = Reflect.ownKeys(READABLE_TRANSFER_TYPES)
+export const TRANSFER_TYPES_LABELS = Reflect.ownKeys(
+  READABLE_TRANSFER_TYPES
+).map(type => READABLE_TRANSFER_TYPES[type])
```

Comment:
Rather than having to do this, we can instead use an array of tuples and work on that instead.

E.g.

```js
const AVAILABLE_TRANSFER_TYPES = [
  [All, 'All'], [Incoming, 'Incoming'], [Outgoing, 'Outgoing']
]

export const TRANSFER_TYPES = AVAILABLE_TRANSFER_TYPES.map(([type]) => type)
export const TRANSFER_TYPES_LABELS = AVAILABLE_TRANSFER_TYPES.map(([_, label]) => label)
```

Reply:
Looks way cleaner!